UI_TESTING.md

```
@@ -0,0 +1,259 @@
+
+# Start development server
+npm run dev
+
+# Frontend will be available at http://localhost:5173 (Vite default)
+```
+
+## Frontend Features to Test
+
+Based on the `frontend/` directory structure, the UI should include:
+
+### 1. Home Page
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)